tool/site/client/tests/components/prescription-summary/props.tsx

```diff
@@ -54,6 +55,9 @@ export const summaryPrescription: PrescriptionSummaryViewProps = {
   patient: summaryPatient,
   practitionerRole: summaryPractitionerRole,
   prescriptionLevelDetails: prescriptionLevelDetailProps,
+  currentPage: 1,
+  pageCount: 1,
+  onPageChange: page => console.log(page),
```

Comment:
just thinking about keeping the console clear could we not just use a void function here? `page => void(page)`